sdk/maps/azure-maps-traffic/src/main/java/com/azure/maps/traffic/models/TrafficIncidentViewport.java

```diff
@@ -14,14 +16,14 @@
      * Viewport Response object
      */
     @JsonProperty(value = "viewpResp", access = JsonProperty.Access.WRITE_ONLY)
-    private TrafficIncidentViewportViewpResp viewpResp;
+    private TrafficIncidentViewportResponse viewpResp;
 
     /**
      * Get the viewpResp property: Viewport Response object.
      *
      * @return the viewpResp value.
      */
-    public TrafficIncidentViewportViewpResp getViewpResp() {
+    public TrafficIncidentViewportResponse getViewpResp() {
```

Comment:
I would change the name of this method in a customization to getViewportResponse()